js/components/interface/menu/MenuSingleItem.js

```diff
@@ -32,125 +33,85 @@ class MenuSingleItem extends React.Component {
     }
   };
 
-  render () {
+  renderArrow () {
+    const { customArrow } = this.props;
+    return customArrow
+      ? customArrow
+      : <i className="fa fa-chevron-right" style={{
+        marginLeft: '10px',
```

Comment:
this styling should be removed but leaving for now to not break vfb and other applications that may rely on it.

Reply:
don't agree here, this style is here to give a default arrow, if the user/developer wants something different they can define a custom arrow with custom style and that will address the problem.

Reply:
I'm ok with the default arrow, I mean that hardcoded 10px margin for instance is not a good default. The default styling should come from the material theme instead.